modules/data-streams/src/main/java/org/elasticsearch/datastreams/DataStreamFeatures.java

```diff
@@ -30,6 +30,10 @@ public class DataStreamFeatures implements FeatureSpecification {
 
     public static final NodeFeature FAILURE_STORE_IN_LOG_DATA_STREAMS = new NodeFeature("logs_data_streams.failure_store.enabled");
 
+    public static final NodeFeature DOWNSAMPLE_INDEX_LEVEL_DEFAULT_METRIC = new NodeFeature(
```

Comment:
Given that this is a downsample feature. I think we should add this feature to the downsample module? It doesn't seem to have a class extending `FeatureSpecification` yet, but I think we can add that.